### PR TITLE
Fixing Netlify Deploy Preview

### PR DESCRIPTION
**Pull request by erikaheidi**
### What should this PR do?
This PR sets additional parameters to the Netlify configuration file to fix the deploy preview. It was not necessary to change the configuration of the website.

### How should this PR be tested?
We should check the Netlify preview to see if it was fixed, and make sure the production site keeps working as expected once changes are incorporated.

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -6,13 +21,13 @@
     X-Content-Type-Options = "nosniff"
     Content-Security-Policy = '''
        default-src 'self';
-       frame-src 'self' https://player.vimeo.com https://www.youtube.com www.youtube-nocookie.com https://platform.twitter.com https://syndication.twitter.com https://visualization-ui.chainguard.app;
-       style-src 'self' 'unsafe-inline' cdn.jsdelivr.net https://fonts.googleapis.com *.netlify.app;
+       frame-src 'self' *.chainguard.dev https://player.vimeo.com https://www.youtube.com www.youtube-nocookie.com https://platform.twitter.com https://syndication.twitter.com https://visualization-ui.chainguard.app;
```
Comment (ltagliaferri) [inline netlify.toml:24]: Can we replace wildcard `*` with `edu` throughout?

File: config/_default/config.toml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-baseurl = "https://edu.chainguard.dev/"
+baseUrl = "https://edu.chainguard.dev/"
```
Comment (ltagliaferri) [inline config/_default/config.toml:1]:
```suggestion
baseURL = "https://edu.chainguard.dev/"
```

On Hugo config I am seeing this

File: config/_default/config.toml [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
 canonifyURLs = false
```
Comment (ltagliaferri) [inline config/_default/config.toml:2]: I am working on analytics related to canonifying URLs, but we may want to revisit documentation here

https://gohugo.io/content-management/urls/#canonicalization

Comment (erikaheidi) [reply config/_default/config.toml:2]: yeah I thought this should be set to true only for production, however I saw this on the docs: `In the May 2014 release of Hugo v0.11, the default value of canonifyURLs was switched from true to false, which we think is the better default and should continue to be the case going forward`

so I don't know anymore. In any case if we decide to change it I think it should go on the config/production folder

**Review by jamonation — APPROVED** [review]
Looks good to me on the preview site, all the assets load now right from netlify.

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**